src/rdkafka.h

```diff
@@ -387,6 +387,8 @@ typedef enum {
         RD_KAFKA_RESP_ERR__APPLICATION = -143,
         /** Assignment lost */
         RD_KAFKA_RESP_ERR__ASSIGNMENT_LOST = -142,
+        /** Aborted */
+        RD_KAFKA_RESP_ERR__ABORTED = -141,
```

Comment:
this name is intentionally general enough to potentially be useful in another context.

oh, i forgot to add description strings etc., additional commit coming.